### ISSUE TITLE
CSS comment partially eaten

### ISSUE THREAD

**Issue by beauroberts** [prettier/prettier#2468]
Apologies if this is the same issue as #2127 - I noticed that in that case the entire comment is eaten, whereas here it's just a substring within the comment.

### Input

```scss
.first-selector, // some|comment
.second-selector {
    color: white;
}
```

### Expected

```scss
.first-selector,
// some|comment
.second-selector {
    color: white;
}
```

### Output

Notice that the part of the comment up to and including the vertical bar is discarded

```scss
.first-selector,
// comment
.second-selector {
    color: white;
}
```

### Repro

https://prettier.io/playground/#%7B%22content%22%3A%22.first-selector%2C%5Cn%2F%2F%20some%7Ccomment%5Cn.second-selector%20%7B%5Cn%20%20%20%20color%3A%20white%3B%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22postcss%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D

Thanks for an amazing tool! We have an enormous codebase and it's been nearly flawless.

**Comment by azz** [comment]
Thanks for the bug report.

This is a bug (or potentially unsupported feature) of one in our dependencies. I've raised it at https://github.com/postcss/postcss-selector-parser/issues/112.

**Comment by suchipi** [comment]
As a workaround, you can use inline comments (normal CSS syntax), but it's still not very pretty:

**Prettier 1.9.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6AZgSwE4GcYC0ucANnGDBNgDQAEA9AFS24QC2cAPpGx7LY3oAdKKmKQoAEyKlylbLWAjaK2pBJUktAO4ALTPADcIgL4hqICAAcYmaLmSgAhtmwRtABRcIHKJwDcITElzECd8ZHQnEmILACNsJzAAazgYAGUrJMwoAHNkGGwAVzgLSQgwSOjYkBzibBgPRNy2JyqY0pAAK1wADwAhRJS09KcOABkcuHaaiCKYK3mAJhnOrLw4bGQQMlwHCytsHJgAdWCYXWQADgAGA7diE8SrbcO4ev9pi2w4AEcinBwJpOFptJBRDoWYhsTAFYqdXA5XJkACKRQg8FWFhgTjiZ0kF2QS2xiUwJCRAGF2K1tlBoF8QEViAAVXG+CHEEwmIA)
```sh
--parser less
```

**Input:**
```less
.first-selector, /* some|comment */
.second-selector {
    color: white;
}
```

**Output:**
```less
.first-selector,
/* some|comment */ .second-selector {
  color: white;
}

```

**Comment by azz** [comment]
Fixed in #3649
